_posts/2024-03-02-social-integration.md

```diff
@@ -102,13 +102,19 @@ An additional challenge was that we release for three OSes: Windows, Linux and M
 And they all deal with plugins in a slightly different way; enough to be challenging.
 A lot of testing, failing, trying again went into getting this right.
 
-Which also meant we had to make choices: get something to work now, and build on it later, or try to do everything at once, potentially never finishing it at all.
+Complicating the situation further, is that we don't have an active MacOS developer, nor do we have access to an actual Mac.
```

Comment:
Please consider this sentence rewording: "Complicating the situation further is the fact that we don't have an active MacOS developer nor an actual Mac." In this sentence you show that you do not have a physical (actual) Mac. In a later sentence you show that you now have access to one via remote communication.